pybricksdev/compile.py

```diff
@@ -113,7 +113,8 @@ async def compile_multi_file(path: str, abi: int):
     """
 
     # compile files using Python to find imports contained within the same directory as path
-    finder = ModuleFinder([os.path.dirname(path)])
+    searchpath = [os.path.dirname(path)]
```

Comment:
```suggestion
    search_path = [os.path.dirname(path)]
```

I have a really hard time reading words that aren't separated.